envergo/templates/haie/moulinette/result_base.html

```diff
@@ -18,7 +18,9 @@
   <hr class="separator fr-mb-3w" />
   <ul class="fr-mb-3w">
     {% for field in moulinette.main_form %}
-      <li>{% field_summary field %}</li>
+      {% block main_form_field %}
```

Comment:
Ah j'avais jamais vu qu'on pouvait mettre un block dans un for ! merci pour la découverte :)